src/js/plugins/tts/plugin.tts.js

```diff
@@ -5,6 +5,7 @@
 import FestivalTTSEngine from './FestivalTTSEngine.js';
 import WebTTSEngine from './WebTTSEngine.js';
 import { toISO6391, approximateWordCount } from './utils.js';
+import { en as tooltips } from './tooltips.js';
```

Comment:
How would you change the language with this approach?

Reply:
I didn't want to tackle adding in a language select since really it's a separate feature, but since I was touching the tooltips and there's been a comment suggesting moving them out for easier i18n [here](https://github.com/internetarchive/bookreader/blob/master/src/js/BookReader.js#L2845), I would start with this.